plugins/mcreator-localization/help/default/particle/render_type.md

```diff
@@ -2,4 +2,4 @@ This parameter defines how this particle should be rendered:
 
 * **Opaque:** Transparent without mipmapping (similar to death particles)
 * **Translucent:** Partially transparent and the most resource heavy option (similar to potion effect particles)
-* **Lit:** Glowing particle that emits light during its lifetime
\ No newline at end of file
+* **Lit:** Glowing particle that emits light during its lifetime (similar to Nether portal particles)
```

Comment:
I don't think Nether portal particles are lit, but explosion particles are for sure. Please verify the Nether portal or change to the explosion.

Reply:
I didn't think about it because I don't like explosions :)
MC code says that lava drips and Nether portal particles are lit, but probably not as bright as explosion ones.